Formula/n/neovim.rb

```diff
@@ -2,13 +2,16 @@ class Neovim < Formula
   desc "Ambitious Vim-fork focused on extensibility and agility"
   homepage "https://neovim.io/"
   license "Apache-2.0"
-
-  head "https://github.com/neovim/neovim.git", branch: "master"
+  revision 1
 
   stable do
+    # TODO: Bump to use tree-sitter 0.26+ when new Neovim release supports it
     url "https://github.com/neovim/neovim/archive/refs/tags/v0.11.5.tar.gz"
     sha256 "c63450dfb42bb0115cd5e959f81c77989e1c8fd020d5e3f1e6d897154ce8b771"
 
+    # TODO: remove `head` block when stable supports tree-sitter 0.26+.
+    depends_on "tree-sitter@0.25"
```

Comment:
Per https://github.com/Homebrew/homebrew-core/pull/257820#discussion_r2603153851, it seems like we should be able to use the latest here even on stable?